src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -58,13 +74,21 @@ class SidebarScreen extends Component {
         Timing.start(CONST.TIMING.SIDEBAR_LOADED, true);
 
         if (this.props.isFirstTimeNewExpensifyUser) {
-            // For some reason, the menu doesn't open without the timeout
-            setTimeout(() => {
-                this.toggleCreateMenu();
-
-                // Set the NVP back to false (this may need to be moved if this NVP is used for anything else later)
-                NameValuePair.set(CONST.NVP.IS_FIRST_TIME_NEW_EXPENSIFY_USER, false, ONYXKEYS.NVP_IS_FIRST_TIME_NEW_EXPENSIFY_USER);
-            }, 200);
+            const hasFreePolicy = _.chain(this.props.policies)
+                .some(policy => policy && policy.type === CONST.POLICY.TYPE.FREE && policy.role === CONST.POLICY.ROLE.ADMIN)
+                .value();
+
+            // If user doesn't have any free policies (workspaces) set up, automatically open create menu
+            if (!hasFreePolicy) {
+                // For some reason, the menu doesn't open without the timeout
+                setTimeout(() => {
+                    this.toggleCreateMenu();
+                }, 200);
```

Comment:
🤮 we should figure out how to do this without a `setTimeout` these hacks end up biting us later down the road because anything else that depends on this code then will have to use a timeout/delay to push it's rendering to the end of the stack.

Reply:
This `setTimeout` is currently our only solution for auto-opening the global create on load, I made [this issue](https://github.com/Expensify/App/issues/5296) for people to look into solutions - read the discussions, it's not a straightforward solutions sadly :D

Reply:
I wouldn't say one comment from a contributor is an "exhaustive" discussion on how best to fix this 😅 but I do agree with Rory's comment so did we ever follow up with react-navigation on this?

Reply:
LOL I meant this issue: https://github.com/Expensify/App/issues/5303 :D